vanilla-tests/src/main/java/fixtures/streamstyleserialization/models/Basic.java

```diff
@@ -108,9 +112,11 @@ public static Basic fromJson(JsonReader jsonReader) {
                         if ("id".equals(fieldName)) {
                             id = JsonUtils.getNullableProperty(reader, r -> reader.getIntValue());
                         } else if ("name".equals(fieldName)) {
-                            name = reader.getStringValue();
+                            name = JsonUtils.getNullableProperty(reader, r -> reader.getStringValue());
```

Comment:
Can it be simplified to `reader.getStringValue()` since string is nullable already?